notebooks/20220512_explore_16s.ipynb

```diff
@@ -0,0 +1,422 @@
+{
```

Comment:
<p>Is this an acronym all micro people will know? If not, might be worth defining in the notebook. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/2022-cf-sputum/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes! it stands for amplicon sequence variant, and is bandied about all over the place. It really is a horrible abbreviation</p><p> </p>